src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -496,15 +500,34 @@ var DynamicTable = new Class({
         this.selectedRows.empty();
     },
 
-    selectRow: function(rowId) {
-        this.deselectAll();
+    selectRow: function(rowId, clear) {
+        if (clear)
+            this.deselectAll();
         this.selectedRows.push(rowId);
+        this.setRowClass();
+        this.onSelectedRowChanged();
+    },
+
+    deselectRow: function(rowId) {
+        this.selectedRows.erase(rowId);
+        this.setRowClass();
+        this.onSelectedRowChanged();
+    },
+
+    selectRows: function(rowIdStart, rowIdEnd) {
+        this.deselectAll();
+
+        var select = false;
+        var that = this;
         this.tableBody.getElements('tr').each(function(tr) {
-            if (tr.rowId == rowId)
-                tr.addClass('selected');
-            else
-                tr.removeClass('selected');
+            if (tr.rowId == rowIdStart)
+                select = !select;
```

Comment:
I would assign value explicitly. And below too.

Reply:
just move `var select` in here.

Reply:
The logic has been changed slightly